doc/sphinx/practical-tools/utilities.rst

```diff
@@ -310,6 +316,16 @@ file timing data:
        (which are frequently noise); lexicographic sorting forces an order on
        files which take effectively no time to compile.
 
+       If you prefer a different sorting order, you can pass
+       ``TIMING_SORT_BY=absolute`` to sort by the total time taken, or
+       ``TIMING_SORT_BY=diff`` to sort by the signed difference in
+       time.
+
+    .. note::
+       Just like ``pretty-timed``, this table defaults to using user
+       times and supports ``TIMING_REAL=1`` to specify that real times
+       rather than user times should be used.
```

Comment:
```suggestion
```

Reply:
This leaves the previous sentence as a fragment.  Did you mean to suggest just deleting "rather than user times"?